### PR TITLE
Static

### PR DESCRIPTION
**Pull request by romaoksenyuk**

### CODE REVIEW

File: api/settings/base.py [hunk 1]
```diff
@@ -145,6 +146,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/dev/howto/static-files/
 
+STATIC_ROOT = "/opt/docker_polls_group/api/static/"
```
Comment (pavlo-mk) [inline api/settings/base.py]: please use `BASE_DIR, static`
BASE_DIR = /opt/docker_polls_group/api/
BASE_DIR is folder where we have manage.py.

**Review by pavlo-mk — CHANGES_REQUESTED** [review]
Please remove from PR all files `api/static/`

File: api/settings/base.py (hunk 1 above)
Comment (m-pasha) [reply api/settings/base.py]: @romaoksenyuk 
STATIC_URL = url for found some static files
STATIC_ROOT = all files will be copied when you make on server `collectstatic` command (in Prod)
STATICFILES_DIRS = if you run `collectstatic` command django found files from paths and copy to STATIC_ROOT folder (in Prod)
[DOCS](https://docs.djangoproject.com/en/2.2/howto/static-files/).

**Comment by pavlo-mk** [comment]
@romaoksenyuk 
STATIC_URL = url for found some static files
STATIC_ROOT = all files will be copied when you make on server `collectstatic` command (in Prod)
STATICFILES_DIRS = if you run `collectstatic` command django found files from paths and copy to STATIC_ROOT folder (in Prod)
[DOCS](https://docs.djangoproject.com/en/2.2/howto/static-files/).

File: api/settings/base.py [hunk 2]
```diff
@@ -2,10 +2,9 @@
 import os
 import sys
 
-
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-sys.path.append(os.path.join(BASE_DIR, 'apps/'))
+sys.path.append(os.path.join(BASE_DIR, 'static'))
```
Comment (pavlo-mk) [inline api/settings/base.py]: Where you will use this path?

File: api/settings/base.py [hunk 3]
```diff
@@ -148,9 +150,22 @@
...
+MEDIA_URL = '/media/'
+
+
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/dev/howto/static-files/
 
+STATIC_ROOT = os.path.join(BASE_DIR, 'allstatic')
 STATIC_URL = '/static/'
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'static'),
```
Comment (pavlo-mk) [inline api/settings/base.py]: Its wrong practice use `os.path.join` in this settings. Please check docs and use simple path `STATIC_ROOT = /allstatic/` If folder put in BASE_DIR you can use simple path without `os.path.join`.

**Review by pavlo-mk — CHANGES_REQUESTED** [review]

**Review by pavlo-mk — APPROVED** [review]
Please make merge.

**Comment by pavlo-mk** [comment]
@klekhaav please approve if correct for you.

**Review by klekhaav — APPROVED** [review]

**Pull request merged**